actions/class.PropertiesAuthoring.php

```diff
@@ -617,13 +617,17 @@ private function invalidatePropertyCache(
             && ($validator->isRangeChanged($currentProperty, $oldProperty)
             || $validator->isPropertyTypeChanged($currentProperty, $oldProperty))
         ) {
-            $this->getParentPropertyListCachedRepository()->deleteCache(
-                [
-                    'propertyUri' => $currentProperty->getUri(),
-                    'listUri' => $oldProperty->getRangeUri()
-                ]
-            );
+            $listUri = $oldProperty->getRangeUri();
+        }
+        if (empty($listUri) && $currentProperty->getRange() === null) {
```

Comment:
Remember this blank lines, please :D 

```suggestion
        }
        
        if (empty($listUri) && $currentProperty->getRange() === null) {
```